src/components/library-item/library-item.jsx

```diff
@@ -9,6 +9,7 @@ import classNames from 'classnames';
 import bluetoothIconURL from './bluetooth.svg';
 import internetConnectionIconURL from './internet-connection.svg';
 
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
It looks to me like this could be converted to a function pretty easily. Any reason not to?